test/schemas.ts

```diff
@@ -1616,6 +1616,14 @@ export const v76Schema = {
   },
 }
 
+export const v77Schema = {
```

Comment:
you don't need a new migration to augment the screens enum

Reply:
@kathaypacific I was wondering about that. I only added it because `test/RootStateSchema.json` was changed.

Am I ok to just remove all the schema update stuff, and only commit the updated `RootStateSchema.json`?

Reply:
yes - the migrations are to safely migrate persisted state to a new app version. if we're augmenting an enum, it's safe for users with persisted state so we don't need a migration :)

Reply:
OK, I removed the migration! I just want to make sure I understand though - so Redux enums are not persisted at all, and pulled from the code every time the app is started - is that correct?